interfaces/builtin/basedeclaration.go

```diff
@@ -307,6 +307,12 @@ slots:
         - app
     deny-connection: true
     deny-auto-connection: true
+  login-control:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+    deny-connection: true
```

Comment:
deny-connection shouldn't be used for implicit core interfaces, please drop it. For future reference, see the comments at the top of basedeclaration.go for how to craft your declaration.